Common/Data/Language/ToDo.TXT

```diff
@@ -417,15 +417,25 @@ _@M2322_ "Enable Radar"
 +_@M2191_ "Cyl Dist"
 
 
-_@M2336_
-ICAO Code 
 
-_@M2252_ "MANUAL"
-_@M2253_ "AUTO THERMALLING"
-_@M2254_ "FULL AUTO"
```

Comment:
why remove this ?

Reply:
Did I?
Sorry, seems to be removed accidentally!